app/javascript/react/components/home-page/root/index.jsx

```diff
@@ -0,0 +1,9 @@
+import React, { useState } from 'react';
+
+function HomePageRoot() {
+  const [welcome, setWelcome] = useState('Welcome to Ecommerce Site!')
```

Comment:
assuming this is just to test and evaluate `useState`?

Reply:
Yup! Decided to leave it in there because I intend to use hooks throughout the project